src/pumipic_lb.hpp

```diff
@@ -9,7 +9,8 @@
 #include <engpar.h>
 #include <particle_structs.hpp>
 
-namespace {
+namespace pumipic {
```

Comment:
Usually the 'anonymous' namespace is used to restrict access to functions/classes/etc. to the file scope.  Was there a specific need for `Parts` and `PartsHash` to be in the `pumipic` namespace?

Reply:
It does not affect any of the tests passing, but I did it to resolve this warning: `pumi-pic/src/pumipic_lb.hpp:33:9: warning: 'pumipic::ParticleBalancer' has a field 'pumipic::ParticleBalancer::sbar_ids' whose type uses the anonymous namespace`